.github/workflows/buildAndTestCMake.yml

```diff
@@ -71,12 +71,9 @@ jobs:
       run: |
         ./build_tools/github_actions/ci_build_cmake_llvm.sh "$LLVM_PROJECT_DIR" "$LLVM_BUILD_DIR"
 
-    - name: Build and Test StableHLO
+    - name: Build and Test StableHLO (with Python bindings)
       shell: bash
       run: |
         ./build_tools/github_actions/ci_build_cmake.sh "$LLVM_BUILD_DIR" "$STABLEHLO_BUILD_DIR"
-
-    - name: Build and Test StableHLO Python API
-      shell: bash
-      run: |
-        ./build_tools/github_actions/ci_build_cmake_python_api.sh "$LLVM_PROJECT_DIR" "$STABLEHLO_PYTHON_BUILD_DIR" "$GITHUB_WORKSPACE"
+        cd "$STABLEHLO_BUILD_DIR"
```

Comment:
Consider moving these commands to the ci_build_cmake script.

Reply:
Turns out these lines aren't even needed.

The 'check-stablehlo-ci' command which the script runs already has registered to it the `check-stablehlo-python` target.

See here:
(look for line that says "python bindings seem to work" at the bottom)

```
❯ cmake --build ./build --target check-stablehlo-ci
[0/92] Running the linalg/ suite

Testing Time: 0.86s

Total Discovered Tests: 7
  Passed: 7 (100.00%)
[85/92] Running the tosa/ suite

Testing Time: 0.06s

Total Discovered Tests: 4
  Passed: 4 (100.00%)
[88/92] cd /usr/local/google/home/fmzakari/code/github.com/openxla/stablehlo/stablehlo/integrations/python/tests && /opt/clion-2022.3/bin/cmake/linux/x64/bin/cmake -E env PYTHONPATH=/usr/local/google/home/fmzakari/code/github.com/openxla/stablehlo/build/python_packages/stablehlo /usr/bin/python3.11 smoketest.py
StableHLO Python bindings seem to work
```

I'll remove these lines.